src/App.js

```diff
@@ -1,23 +1,57 @@
 import React from 'react';
```

Comment:
```suggestion
import React, { Component } from 'react';
```
вот теперь заработает:)